cargo-dist/src/init.rs

```diff
@@ -598,6 +600,13 @@ fn update_toml_metadata(
         installers.as_ref(),
     );
 
+    apply_optional_value(
+        table,
+        "tap",
+        "# A GitHub repo to push the Homebrew formula to\n",
```

Comment:
```suggestion
        "# A GitHub repo to push Homebrew formulas to\n",
```

(pedantically more accurate I guess)